components/geometry-merger/examples/buffer/index.html

```diff
@@ -4,17 +4,27 @@
     <meta charset="utf-8">
```

Comment:
Wasn't sure about keeping this example, now all examples use BufferGeometry.  It's a little different from the basic example, so I decided to keep it.

But it's not *that* interesting (doesn't really add anything over the basic example) so I didn't correct the fact that it is *not* included in the [index page](https://github.com/supermedium/superframe/blob/master/components/geometry-merger/index.html) for the component.